packages/engine-test-utils/src/__tests__/pods-core/MarkdownPod.spec.ts

```diff
@@ -503,11 +505,21 @@ describe("markdown import pod", () => {
             encoding: "utf8",
           }
         );
+        const fileBody3Content = fs.readFileSync(
+          path.join(vpath, "folder.something.md"),
+          {
+            encoding: "utf8",
+          }
+        );
         expect(fileBody.match("test.txt")).toBeTruthy();
         const assetPath = path.join("assets", "test.txt").replace(/[\\]/g, "/");
+        const imagePath = path
+          .join("assets", "image.JPEG")
+          .replace(/[\\]/g, "/");
         expect(fileBodyContent).toContain(`[test-pdf](/${assetPath})`);
         expect(fileBody2Content).toContain(`[test-pdf](/${assetPath})`);
-        expect(assetsDir.length).toEqual(3);
+        expect(fileBody3Content).toContain(`[image](/${imagePath})`);
```

Comment:
This test doesn't seem to test the linked issue. In the linked issue, the markdown file contains a link where the asset is in the same directory as the note, and the link looks like `![](image.jpg)`. The link is relative from the note file itself and not the root folder we're importing from.